src/logistration/RegistrationPage.jsx

```diff
@@ -324,7 +324,6 @@ RegistrationPage.defaultProps = {
   },
 };
 
-
 RegistrationPage.propTypes = {
```

Comment:
Are the extra lines you removed in various files was because linter was reporting it?

Reply:
@adeelkhan yes.

Reply:
Strange. This can happen for old code before linter, but how new code got merged.

Reply:
I have changed a few rules in `.eslintrc.js` that might be the reason.

Reply:
I have changed the indent to 2 from the default 4, that's why it's restricting max empty lines to one.

Reply:
Not part of the scope :)